teamcity: failed tests on master: lint/TestLint, lint/TestLint: TestLint/TestErrCheck, lint/TestLint: TestLint/TestReturnCheck, lint/TestLint: TestLint/TestMegacheck, lint/TestLint, lint/TestLint: TestLint/TestErrCheck, lint/TestLint: TestLint/TestReturnCheck, lint/TestLint: TestLint/TestMegacheck

The following tests appear to have failed:

[#536341](https://teamcity.cockroachdb.com/viewLog.html?buildId=536341):


```
--- FAIL: lint/TestLint (18.910s)
None
--- FAIL: lint/TestLint: TestLint/TestErrCheck (8.150s)
lint_test.go:902: err=exit status 2, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/storage/allocator_test.go:1300:4: cannot convert nil (untyped nil value) to github.com/cockroachdb/cockroach/pkg/config.ZoneConfig
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/allocator_test.go:1369:4: cannot convert nil (untyped nil value) to github.com/cockroachdb/cockroach/pkg/config.ZoneConfig
	error: failed to check packages: could not type check: couldn't load packages due to errors: github.com/cockroachdb/cockroach/pkg/storage
--- FAIL: lint/TestLint: TestLint/TestReturnCheck (10.760s)
lint_test.go:929: err=exit status 1, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/storage/allocator_test.go:1300:4: cannot convert nil (untyped nil value) to github.com/cockroachdb/cockroach/pkg/config.ZoneConfig
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/allocator_test.go:1369:4: cannot convert nil (untyped nil value) to github.com/cockroachdb/cockroach/pkg/config.ZoneConfig
--- FAIL: lint/TestLint: TestLint/TestMegacheck (14.530s)
lint_test.go:990: couldn't load packages due to errors: github.com/cockroachdb/cockroach/pkg/storage
------- Stdout: -------
=== PAUSE TestLint/TestMegacheck
--- FAIL: lint/TestLint (18.910s)
None
--- FAIL: lint/TestLint: TestLint/TestErrCheck (8.150s)
lint_test.go:902: err=exit status 2, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/storage/allocator_test.go:1300:4: cannot convert nil (untyped nil value) to github.com/cockroachdb/cockroach/pkg/config.ZoneConfig
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/allocator_test.go:1369:4: cannot convert nil (untyped nil value) to github.com/cockroachdb/cockroach/pkg/config.ZoneConfig
	error: failed to check packages: could not type check: couldn't load packages due to errors: github.com/cockroachdb/cockroach/pkg/storage
--- FAIL: lint/TestLint: TestLint/TestReturnCheck (10.760s)
lint_test.go:929: err=exit status 1, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/storage/allocator_test.go:1300:4: cannot convert nil (untyped nil value) to github.com/cockroachdb/cockroach/pkg/config.ZoneConfig
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/allocator_test.go:1369:4: cannot convert nil (untyped nil value) to github.com/cockroachdb/cockroach/pkg/config.ZoneConfig
--- FAIL: lint/TestLint: TestLint/TestMegacheck (14.530s)
lint_test.go:990: couldn't load packages due to errors: github.com/cockroachdb/cockroach/pkg/storage
------- Stdout: -------
=== PAUSE TestLint/TestMegacheck

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/allocator_test.go
+++ b/pkg/storage/allocator_test.go
@@ -1297,7 +1297,7 @@ func TestAllocatorRebalanceDifferentLocalitySizes(t *testing.T) {
 	for i, tc := range testCases {
 		result, details := a.RebalanceTarget(
 			ctx,
-			nil, /* constraints */
+			config.ZoneConfig{},
 			nil, /* raftStatus */
 			testRangeInfo(tc.existing, firstRange),
 			storeFilterThrottled,
@@ -1366,7 +1366,7 @@ func TestAllocatorRebalanceDifferentLocalitySizes(t *testing.T) {
 		log.Infof(ctx, "case #%d", i)
 		result, details := a.RebalanceTarget(
 			ctx,
-			nil, /* constraints */
+			config.ZoneConfig{},
 			nil, /* raftStatus */
 			testRangeInfo(tc.existing, firstRange),
 			storeFilterThrottled,

